org.sparkleshare.SparkleShare.json

```diff
@@ -0,0 +1,136 @@
+{
+    "app-id": "org.sparkleshare.Sparkleshare",
+    "branch": "master",
+    "command": "sparkleshare",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.24",
+    "sdk": "org.gnome.Sdk",
+    "metadata": "metadata",
+    "separate-locales": false,
+    "finish-args": [
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=system-bus",
+        "--socket=session-bus",
+        "--talk-name=org.gtk.vfs",
+        "--talk-name=org.gtk.vfs.*",
+        "--share=ipc",
+        "--device=dri",
+        "--share=network",
+        "--filesystem=home"
+    ],
+    "modules": [
+        {
+            "name": "mono",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://download.mono-project.com/sources/mono/mono-4.2.2.30.tar.bz2",
+                    "sha256": "57858cd033be9915d7abdc5158c1faae8fa05757c3b7117cab3d703aa696c56b"
+                }
+            ]
+        },
+        {
+            "name": "openssh",
+            "config-opts": ["--without-pie"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://mirror.exonetric.net/pub/OpenBSD/OpenSSH/portable/openssh-7.5p1.tar.gz",
+                    "sha256": "9846e3c5fab9f0547400b4d2c017992f914222b3fd1f8eee6c7dc6bc5e59f9f0"
+                }
+            ]
+        },
+        {
+            "name": "git",
+            "config-opts": ["--with-openssl=no"],
```

Comment:
Why without openssl?

Reply:
Couldn't get it to compile otherwise and it does the job without.

Reply:
This seems to work now. I think the earlier platforms didn't have openssl included.